packages/x-codemod/src/v6.0.0/pickers/replace-toolbar-props-by-slot/actual.spec.js

```diff
@@ -0,0 +1,7 @@
+<DatePicker ToolbarComponent={MyToolbar} />;
```

Comment:
Try adding this example as well. It reveals a somewhat ugly transformation result of:
```tsx
componentsProps={{
  toolbar: {
    hidden: !false,
  },
}}
```
Maybe we could make sure such case would end up as `hidden: true`? 🤔 
```suggestion
<DatePicker ToolbarComponent={MyToolbar} showToolbar={false} />;
```